graph/src/KokkosGraph_Distance1ColorHandle.hpp

```diff
@@ -558,11 +558,11 @@ class GraphColoringHandle {
    */
   void set_defaults(const ColoringAlgorithm &col_algo) {
     switch (col_algo) {
-      case COLORING_VB:
-      case COLORING_VBBIT:
-      case COLORING_VBCS:
-      case COLORING_VBD:
-      case COLORING_VBDBIT:
+      case COLORING_VB: [[fallthrough]];
```

Comment:
@cwpearson Does this situation trigger the warning? I wasn't familiar with this fallthrough attribute before, but I looked at this example:

https://en.cppreference.com/w/cpp/language/attributes/fallthrough (between cases 1 and 2)

and it doesn't say there will be a warning if multiple case labels point to the same statement.